### PR TITLE
SEO improvements

### PR DESCRIPTION
**Pull request by dcyar**
- Correction of image and link titles
- Correction of titles (only one H1 tag per page)

### CODE REVIEW

File: src/pages/blog/[...page].astro [hunk 1]
```diff
@@ -18,7 +18,7 @@ const { page } = Astro.props;
 ---
 
 <Base meta_title="Blog">
-  <h1 class="text-2xl md:text-3xl font-bold mb-8">Blog</h1>
+  <h2 class="text-2xl md:text-3xl font-bold mb-8">Blog</h2>
```
Comment (gndx) [inline src/pages/blog/[...page].astro:21]: What is the proposal for changing h1 to h2?

File: src/layouts/Page.astro [hunk 2]
```diff
@@ -3,6 +3,6 @@ const { title } = Astro.props;
 ---
 
 <div class="prose prose-green prose-md md:prose-lg min-w-full">
-  <h1 class="text-3xl font-bold">{title}</h1>
+  <h2 class="text-3xl font-bold">{title}</h2>
```
Comment (gndx) [inline src/layouts/Page.astro:6]: What is the proposal for changing h1 to h2?

File: src/pages/categories/index.astro [hunk 3]
```diff
@@ -8,7 +8,7 @@ const categories = await getTaxonomy("blog", "categories");
 
 <Base>
   <section class="grid grid-cols-1">
-    <h1 class="text-2xl md:text-3xl font-bold mb-8">Categories:</h1>
+    <h2 class="text-2xl md:text-3xl font-bold mb-8">Categories:</h2>
```
Comment (gndx) [inline src/pages/categories/index.astro:11]: What is the proposal for changing h1 to h2?

File: src/components/Header.astro [hunk 4]
```diff
@@ -18,11 +18,12 @@ const { main }: { main: NavigationLink[] } = menu;
 <header class="bg-zinc-50 dark:bg-zinc-900">
   <div class="mx-auto max-w-screen-lg px-4 py-8 sm:px-6 sm:py-12 lg:px-8">
     <div class="flex items-start justify-between gap-2 sm:gap-4 md:items-center">
-      <a href="/" class="logo flex items-center" aria-current="page">
+      <a href="/" class="logo flex items-center" aria-current="page" title={config.site.title}>
```
Comment (gndx) [inline src/components/Header.astro:21]: Is it necessary to include a 'title' attribute in all <a> elements, Can you provide any documentation regarding the importance of using the 'title' attribute?

**Review by gndx — CHANGES_REQUESTED** [review]
Is it necessary to include a 'title' attribute in all <a> elements, Can you provide any documentation regarding the importance of using the 'title' attribute?

**Comment by dcyar** [comment]
> Is it necessary to include a 'title' attribute in all elements, Can you provide any documentation regarding the importance of using the 'title' attribute?

I found a reference to the title attribute in this blog post from [Google Developers](https://developers.google.com/search/blog/2007/12/using-alt-attributes-smartly), where they mention that the title attribute provides additional information about the element it's configured in. Similarly, on websites of SEO professionals or marketing agencies, they indicate that title attributes within elements aid accessibility. This is because screen readers that some users utilize rely on these attributes to provide a better user experience.

Although the title attribute doesn't directly impact SEO, I believe that the same importance should also be given to accessibility.

File: src/pages/blog/[...page].astro (hunk 1 above)
Comment (dcyar) [reply src/pages/blog/[...page].astro:21]: There should only be one H1 header per page, in this case, along with the hero section title and the titles of the pages: Blog, Tags, Categories. Having the H1 header duplicated in these pages would occur.

**Pull request closed without merging**